sbi2-2023-cytotable-poster/poster.html

```diff
@@ -0,0 +1,589 @@
+<h5 id="affiliation"><sup>1</sup> Department of Biomedical Informatics, University of Colorado Anschutz</h5>
+</div>
+<!-- Right Logo  -->
+<div class="logo_right">
+<img src=images/combined.png style="width: 80%">
+</div>
+</div>
+
+<div class='poster_body_wrap'>
+<div class='poster_body'>
+<div id="introduction" class="section level1">
+<h1>Introduction</h1>
+<script type="module">
+  import mermaid from 'https://unpkg.com/mermaid@9/dist/mermaid.esm.min.mjs';
+  mermaid.initialize({ startOnLoad: true });
+</script>
+<center>
+<pre class="mermaid" style="width:70%">
+flowchart LR
+
+    images[(Microscopy\nimages)]
+    csv[(CSV files)]
+    npz[(NPZ files)]
+    sqlite[(SQLite files)]
+    extract["Single-cell\nFeature extraction\n(for ex. CellProfiler)"]
+    csv[(CSV files)]
+    npz[(others ...)]
+    sqlite[(SQLite files)]
+    pycytominer["\nPycytominer\n\n"]
+        
+    subgraph cytotable["CytoTable (Python package)"]
+        subgraph spacer [" "]
+            subgraph parsl ["Data engine: Parsl\n"]
+                subgraph spacer2 [" "]
+                    subgraph sql["Data language: SQL (DuckDB)"]
+                        subgraph spacer3 [" "]
+                            subgraph queryopt["Query optimizer: DuckDB (runs SQL)"]
+                                subgraph spacer4 [" "]
+                                    subgraph representations["Data representations (intermediate and final)"]
+                                        arrow[("Apache Arrow\ntable(s)\n(in-memory)")]
+                                        parquet[("Parquet dataset(s)\n(serialized)")]
+                                    end
+                                end
+                            end
+                        end
+                    end
+                    
+                end
+            end
+        end
+    end
+
+    images --> extract
+    extract --> csv
+    extract --> sqlite
+    extract --> npz
+    csv --> cytotable
+    sqlite --> cytotable
+    npz --> cytotable
+    arrow --> parquet
+    cytotable --> pycytominer
+
+style sql fill:#A7F3D0,stroke:#999999;
+style cytotable fill:#FEF9C3,stroke:#444444;
+style parsl fill:#BAE6FD,stroke:#999999;
+style representations fill:#F0FDFA,stroke:#999999;
+style queryopt fill:#DCFCE7,stroke:#999999;
+style spacer fill:#FEF9C3,stroke:#FEF9C3;
+style spacer2 fill:#BAE6FD,stroke:#BAE6FD;
+style spacer3 fill:#A7F3D0,stroke:#A7F3D0;
+style spacer4 fill:#DCFCE7,stroke:#DCFCE7;
+</pre>
+</center>
+<p class="figcaption">
+Figure 1. Diagram showing high-dimensional single-cell morphology data flow with relationship to CytoTable modular data stack components.
```

Comment:
```suggestion
Figure 1. The flow of high-dimensional single-cell morphology data with relationship to CytoTable modular data stack components.
```

Rather than state that it is a "diagram", just state what it shows.